src/EnergyPlus/ChillerElectricEIR.hh

```diff
@@ -203,7 +203,7 @@ namespace ChillerElectricEIR {
         Real64 CondenserFanEnergyConsumption; // reporting: Air-cooled condenser fan energy [J]
         Real64 BasinHeaterConsumption;        // Basin heater energy consumption (J)
         bool IPLVFlag;
-        bool EquipFlowCtrl;
+        int EquipFlowCtrl;
```

Comment:
Noticed this while compiling. It appears that EquipFlowCtrl is an int but I have to check this again now that diffs are gone.